front/components/vaults/CreateOrEditVaultModal.tsx

```diff
@@ -107,15 +128,6 @@ export function CreateOrEditVaultModal({
           </DataTable.CellContent>
         ),
         enableSorting: false,
-        filterFn: (row: Row<RowData>, columnId: string, filterValue: any) => {
```

Comment:
Note: got rid of the `filterFn` on "name" as we now perform search through the `members/search` endpoint. The search is performed on emails, to not have to deal with diacritics along with sequelize...

Reply:
and names i hope ?

Reply:
@Fraggle for now the search is only performed on the email column for the above mentioned reasons. This decision was taken together with @flvndvd.